google-cloud-pubsub/src/test/java/com/google/cloud/pubsub/v1/SubscriberTest.java

```diff
@@ -205,6 +205,9 @@ public void testAckSingleMessage() throws Exception {
 
   @Test
   public void testGetSubscriptionOnce() throws Exception {
+    if (isStreamingTest) {
```

Comment:
I assume you'll clean up the tests later? since there is no point on keeping the logic to test non streaming polling